DedicatedServer/Scripts/Libs/Zrx/ModeLibs/Common/Commands/ModeCommands.Script.txt

```diff
@@ -1,7 +1,7 @@
 #Include "TextLib" as TL
 #Include "Libs/Zrx/ModeLibs/Common/Commands/CommandUtils.Script.txt" as CommandUtils
 
-#Const C_UIPosition_Y							65.
+#Const C_UIPosition_Y							20.
```

Comment:
```suggestion
#Const C_UIPosition_Y							-50.
```
![image](https://user-images.githubusercontent.com/53338174/148599680-9a625771-e9b2-441c-b9e8-d4b2d6816ff6.png)

Doesn't Conflict with Scoreboard, BigMessages, Header, ControllerUI or FollowFlagButton.
Can only conflict with radar, but it's an admin tool and shouldn't be too big of a deal.